### PR TITLE
X.26

### PR DESCRIPTION
**Pull request by yevgenp**

### CODE REVIEW

**Comment by yevgenp** [comment]
@peter-lawrey please check the deprecations left. There some dependencies thus I was unable to remove them.

File: src/main/java/net/openhft/chronicle/core/util/Longs.java [hunk 1]
```diff
@@ -33,7 +33,6 @@
  * A collection of functional compositions to check and assert various requirements
  * related to {@code long} values.
  */
-@Deprecated(/* to be moved to internal in Chronicle-Bytes in x.26 as that is where it's used. One method is used in UnsafeMemory */)
```
Comment (JerryShea) [inline src/main/java/net/openhft/chronicle/core/util/Longs.java]: if we are going to de-deprecate classes/methods then this change should also be made in another PR and merged ASAP into develop. That way, people won't remove calls to methods which are no longer deprecated

Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/core/util/Longs.java]: done

**Review by tgd — APPROVED** [review]

**Review by peter-lawrey — APPROVED** [review]

File: src/main/java/net/openhft/chronicle/core/Mocker.java [hunk 2]
```diff
@@ -21,11 +21,8 @@
...
 
 import java.io.PrintStream;
-import java.io.PrintWriter;
 import java.io.StringWriter;
 import java.util.concurrent.BlockingQueue;
-import java.util.function.BiConsumer;
-import java.util.function.Consumer;
 
 /**
  * The Mocker class provides utility methods for creating mocked instances of interfaces.
```
Comment (JerryShea) [inline src/main/java/net/openhft/chronicle/core/Mocker.java:28]: should refer to util.Mocker with a comment explaining this here left for backwards compatibility?

**Review by JerryShea — APPROVED** [review]
approved with 1 comment

File: src/main/java/net/openhft/chronicle/core/Mocker.java (hunk 2 above)
Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/core/Mocker.java:28]: I have added a comment
